robMsg/json/de.json

```diff
@@ -134,6 +134,17 @@
 	"LOOP_FOREVER": "Wiederhole unendlich oft",
 	"MENU_ABOUT": "Über das Open Roberta Lab",
 	"MENU_ABOUT_PROJECT": "Über das Open Roberta Projekt",
+	"MENU_SHORTCUT_IMPORT": "Ctrl + I: Programm importieren",
+	"MENU_SHORTCUT_EXPORT": "Ctrl + E: Programm exportieren",
+	"MENU_SHORTCUT_SAVE": "Ctrl + S: Speichern",
+	"MENU_SHORTCUT_SAVEAS": "Ctrl + Shift + S: Speichern als",
```

Comment:
Please use here "Speichern unter" as it is defined in the German translation of the MENU_SAVE_AS key